packages/cli/src/index.ts

```diff
@@ -1,6 +1,7 @@
 import chalk from 'chalk';
 import childProcess from 'child_process';
 import commander from 'commander';
+import didYouMean from 'didyoumean';
```

Comment:
eslint giving this error `Unable to resolve path to module 'didyoumean'  import/no-unresolved`!

Reply:
Linter passing upstream, so going to ignore this and ship it. Will be fixed later if needed, thank you!